crates/biome_js_analyze/src/semantic_analyzers/correctness/use_exhaustive_dependencies.rs

```diff
@@ -524,6 +524,55 @@ fn is_out_of_function_scope(
     )
 }
 
+fn into_member_vec(node: &JsSyntaxNode) -> Vec<String> {
```

Comment:
We should document what this function does because the name isn't very descriptive